wire-schema-tests/src/commonTest/kotlin/com/squareup/wire/recipes/LogToFileHandlerTest.kt

```diff
@@ -54,6 +54,10 @@ class LogToFileHandlerTest {
             |}
         """.trimMargin(),
       )
+      // We manually add fake runtime protos to please Wire when running on a non-JVM platforms.
```

Comment:
Wanna create a helper function that does this, and can contain the explanation doc on it?
```
addFakeRuntimeProtos()
```